### PR TITLE
Fixed Follow Unfollow with checks. Changed also the HTML of /wall/ page

### PR DESCRIPTION
**Pull request by nicolotonci**

### CODE REVIEW

File: monolith/database.py [hunk 1]
```diff
@@ -111,3 +111,6 @@ def store_dice_set(dice_set):
     db_entry.serialized_dice_set = json.dumps(dice_set.serialize())
     db.session.add(db_entry)
     db.session.commit()
+
+def isFollowing(who, by_who):
+    return True if db.session.query(Follow).filter(Follow.followed_by_id == by_who).filter(Follow.user_id == who).count() > 0 else False
```
Comment (sduoase) [inline monolith/database.py]: you can just return db.session.query(Follow).filter(Follow.followed_by_id == by_who).filter(Follow.user_id == who).count() > 0

Comment (nicolotonci) [reply monolith/database.py]: Yes right!

Comment (nicolotonci) [reply monolith/database.py]: Please review the last commit!

**Comment by sduoase** [comment]
Ping for review.

**Pull request merged**